HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -1567,6 +1587,9 @@ def self.create_supp_heating_coil(model, obj_name, heat_pump)
     end
     htg_supp_coil.setNominalCapacity(UnitConversions.convert(capacity, 'Btu/hr', 'W'))
     htg_supp_coil.setName(obj_name + ' ' + Constants.ObjectNameBackupHeatingCoil)
+    if is_ddb_control and (not htg_coil.nil?)
+      apply_supp_control_for_ddb_thermostat(model, htg_supp_coil, control_zone, htg_coil)
+    end
```

Comment:
Is it even possible to have a backup heating coil but no heating coil? (What would it be backup relative to?) It seems to me that `not htg_coil.nil?` is not needed.